cumulus_library/study_manifest.py

```diff
@@ -111,19 +111,29 @@ def get_statistics_file_list(self) -> list[str] | None:
         stats_config = self._study_config.get("statistics_config", {})
         return stats_config.get("file_names", [])
 
-    def get_export_table_list(self) -> list[str] | None:
+    def get_export_table_list(self) -> list[tuple] | None:
```

Comment:
nit: I don't personally love using tuples as data types. They're so opaque when using them and don't play well with type hinting. A namedtuple or dataclass might be a good lightweight option?

Reply:
sure, i can dataclass this.